dash/dash.py

```diff
@@ -346,6 +347,17 @@ class Dash:
     :param hooks: Extend Dash renderer functionality by passing a dictionary of
     javascript functions. To hook into the layout, use dict keys "layout_pre" and
     "layout_post". To hook into the callbacks, use keys "request_pre" and "request_post"
+
+    :param routing_callback_inputs: When using Dash pages (use_pages=True), allows to
+    add new States to the routing callback, to pass additional data to the layout
+    functions. The syntax for this parameter is a dict of State objects:
+    `routing_callback_inputs={"language": Input("language", "value")}`
+    This allows things like (non-exhaustive list):
+    * A language dropdown that will be passed to every layout function,
+      for internationalisation
```

Comment:
While localization might be a use case, this allows for more so we can delete this from the docstring.